File: Java/String-2/doubleChar.java
/*
 * Given a string, return a string where for every char
 * in the original, there are two chars.
 */

public String doubleChar(String str) {
  String doubleChar = "";
  // Add every character in str to doubleChar twice
  for(int i = 0; i < str.length(); i++){
    doubleChar = doubleChar + str.charAt(i) + str.charAt(i);
  }
  return doubleChar;
}

File: Python/String-2/double_char.py
# Given a string, return a string where for every char in the
# original, there are two chars.

def double_char(str):
  double_char = ''
  
  # Add every character into double_char twice
  for i in range(len(str)):
    double_char += 2 * str[i]
  return double_char
